doc/design/dandiset-issues-tracking.md

```diff
@@ -0,0 +1,85 @@
+# Dandiset issues reporting/tracking
+
+Dandisets could have issues related to problems with
+
+- uploaded data files
+- metadata records
+- missing files
+- validation errors
+- fix/enhancement proposals
+- etc.
+
+We need to enable an easy and consistent interface to allow archive users to search, report, followup, signup etc to them.
+Overall, we need an issue tracking system, while avoiding creating one from scratch.
+
+## Current Process
+
+Largely not formalized:
+
+- DLP does not have any pointer to report an issue for Dandisets
+  - footer has overall link to report issues for the archive (and incorrect: [PR to fix](https://github.com/dandi/dandi-archive/pull/1594))
+- We generally refer people to [helpdesk discussions](https://github.com/dandi/helpdesk/discussions) as common place to report issues
+- While preparing DataLad versions of the Dandisets and posting them on GitHub we do not disable `Issues` feature:
+  - for some Dandisets, e.g. https://github.com/dandisets/000108/issues we already used GitHub issues
+  - we do not anyhow automatically add/invite original uploaders to those dandisets.
+    - note: ATM we do not collect audit information on who specifically uploaded any asset to reliably deduce github users to assign.
+- The issue discussing this: https://github.com/dandi/dandi-archive/issues/863
+
+
+## Proposed Process
+
+**Overall**: Reuse existing Dandisets on GitHub (https://github.com/dandisets), and available for them GitHub's issue tracker.
+
+- [x] Enable Issue tracking per Dandiset.
+  - In the process of creating DataLad Dandisets on GitHub we already enable `Issues` functionality for all dandisets.
+- [ ] Initiate documentation in handbook on managing issues of the dandisets
+- [ ] Upon creation of a DataLad Dandiset on GitHub:
+  - Invite all Dandiset owners in Triage role to the dandiset. In Triage scope they can close, label, assign issues. The other actions (e.g. user management) would need to be implemented using [bot].
+  - Email them pointing to documentation (item above)
+- [ ] Setup https://github.com/dandisets [organization-wide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file) [`.github/ISSUE_TEMPLATE`](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) with different templates/types of issues to report
```

Comment:
Do you mean that the [dandisets org](https://github.com/dandisets) would be used across all instances to store all issues? I think it would be better to let each instance configure the org in which it will store its issue repos.